app/workers/vbms/submit_dependents_pdf_job.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module VBMS
+  class SubmitDependentsPDFJob
+    include Sidekiq::Worker
+    include SentryLogging
+
+    # Generates PDF for 686c form and uploads to VBMS
+    def perform(saved_claim_id, veteran_info)
+      claim = SavedClaim::DependencyClaim.find(saved_claim_id)
+      claim.format_and_upload_pdf(veteran_info)
```

Comment:
I think it might be good practice to validate that the `claim` you've found is valid. It's not very likely, but it is possible that the `dependency_claim.rb` file changes in code & gets redeployed, and this job ends up finding an "old" form of the DependencyClaim.

Are there any attributes that you know are mandatory that you could check for first to fail early just in case they aren't there?

Reply:
Yeah, that's a good idea. We will get a list together and see about adding validation.

Reply:
We added some basic validation for the form data getting passed in. LMK if this was what you had in mind.


Link for your convenience:
https://github.com/department-of-veterans-affairs/vets-api/pull/4585/files#diff-a7e4458dfa7cec528b9bdbb7a0cb353eR15